tests/test_functions.py

```diff
@@ -872,3 +872,24 @@ def test_reset_index_inplace_drop(dataframe):
         dataframe.reset_index(drop=True),
         dataframe.reset_index_inplace(drop=True),
     )
+
+        
+def test_select_columns(dataframe):
+    df = dataframe.select_columns(columns= [
```

Comment:
To make the below fix work, I would suggest doing:

```suggestion
    columns = ['a', 'Bell__Chart', 'cities']
    df = dataframe.select_columns(columns=columns)
```